sapi/fpm/tests/bug73342-nonblocking-stdio.phpt

```diff
@@ -0,0 +1,46 @@
+--TEST--
+Bug #73342: Vulnerability in php-fpm by changing stdin to non-blocking
```

Comment:
NIT: Please could you change the name to something like:
```
FPM: bug73342 - Non-blocking stdin
```
so it's a bit more consistent with other FPM bug tests.